tests/functional/simple_seed/test_seed.py

```diff
@@ -1,19 +1,290 @@
+        else:
+            check_table_does_not_exist(project.adapter, "models__downstream_from_seed_actual")
+
+
+class TestBasicSeedTests(SeedTestBase):
+    def test_postgres_simple_seed(self, project):
+        """Build models and observe that run truncates a seed and re-inserts rows"""
+        self._build_relations_for_test(project)
+        self._check_relation_end_state(run_result=run_dbt(["seed"]), project=project, exists=True)
+
+    def test_postgres_simple_seed_full_refresh_flag(self, project):
+        """Drop the seed_actual table and re-create. Verifies correct behavior by the absence of the
+        model which depends on seed_actual."""
+        self._build_relations_for_test(project)
+        self._check_relation_end_state(
+            run_result=run_dbt(["seed", "--full-refresh"]), project=project, exists=False
+        )
+
+
+class TestSeedConfigFullRefreshOn(SeedTestBase):
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {"quote_columns": False, "full_refresh": True},
+        }
+
+    def test_postgres_simple_seed_full_refresh_config(self, project):
+        """config option should drop current model and cascade drop to downstream models"""
+        self._build_relations_for_test(project)
+        self._check_relation_end_state(run_result=run_dbt(["seed"]), project=project, exists=False)
+
+
+class TestSeedConfigFullRefreshOff(SeedTestBase):
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {"quote_columns": False, "full_refresh": False},
+        }
+
+    def test_postgres_simple_seed_full_refresh_config(self, project):
+        """Config options should override full-refresh flag because config is higher priority"""
+        self._build_relations_for_test(project)
+        self._check_relation_end_state(run_result=run_dbt(["seed"]), project=project, exists=True)
+        self._check_relation_end_state(
+            run_result=run_dbt(["seed", "--full-refresh"]), project=project, exists=True
+        )
+
+
+class TestSeedCustomSchema(SeedTestBase):
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        """Create table for ensuring seeds and models used in tests build correctly"""
+        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {
+                "schema": "custom_schema",
+                "quote_columns": False,
+            },
+        }
+
+    def test_postgres_simple_seed_with_schema(self, project):
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # TODO: 1/4 custom schema work; but unsure how to provide these lines with custom schema on seed_actual
```

Comment:
So, I did my best and couldn't quite figure out the right way to tell the test framework a custom schema for a table. Suggestions?

Reply:
Looks like we can define the schema for seed in dbt_project.yml [here](https://docs.getdbt.com/reference/seed-configs#general-configurations-1). You can overwrite it by overwriting [this fixture](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/tests/fixtures/project.py#L134) in your class for now and include the seed definitions. But if you feel like that is something needed in other places also, you can also make an update to that `dbt_profile_data` to make it require another fixture returns seed configurations and add it to the dbt_profile.

Reply:
Bah, I asked this question terribly. That's on me. What I meant to ask was how do I specify a custom schema for `check_relations_equal`. Lo and behold, dive deep enough into our test frame and you find you can specify a custom schema. Problem solved! 👍

Reply:
Awesome!